### PR TITLE
Update to Node v8.4.0

### PR DESCRIPTION
**Pull request by kamranayub**
## Changes:

- Update to Node v8

## Notes

- [x] Ready to merge
- [ ] Wait

Performance of npm and Node itself is a lot better in v8.

### CODE REVIEW

File: GruntFile.js [hunk 1]
```diff
@@ -291,6 +291,17 @@ module.exports = function (grunt) {
...
+      },
+
+      connect: {
+         sandbox: {
+            options: {
+               port: '3001',
+               useAvailablePort: true,
+               keepalive: true,
+               base: './sandbox'
+            }
```
Comment (eonarheim) [inline GruntFile.js]: Nice! Will this keep us from having to spin up a local server every type we want to test sandbox???? 🥇

**Comment by kamranayub** [comment]
Going to remove shrinkpack for now because it cannot be used yet with npm 5. Since package-lock (shrinkwrap.json) and modules are cached on CI, I don't see a big issue living without shrinkpack for awhile.

File: GruntFile.js (hunk 1 above)
Comment (kamranayub) [reply GruntFile.js]: YEE 💯

Comment (kamranayub) [reply GruntFile.js]: And can use to do functional tests if needed.

**Pull request merged**